sources/packages/backend/apps/db-migrations/src/sql/EducationPrograms/Add-col-none-of-entrance-requirements.sql

```diff
@@ -0,0 +1,7 @@
+-- Add none_of_entrance_requirements column to education_programs table
+ALTER TABLE
+    sims.education_programs
+ADD
+    COLUMN none_of_entrance_requirements BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
I believe that the other entrance requirements columns are all accepting NULL. Do we need to change the behavior for this one?